### PR TITLE
Add popup window

### PR DESCRIPTION
**Pull request by ONJoseph**
-add close icon to popup
-add functionality to open popup
-add functionality to menu icon
-add function modal

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -360,4 +360,4 @@ <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
     <script src="main.js"></script>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (sja-thedude) [inline index.html:363]: - [ ] Kindly link your js file correctly. You have placed the main.js file in the src folder so link according.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @ONJoseph, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Great job on working on the popup window with creating the section dynamically.
✔️ Good PR Title and Description


## Required 
- As your JS file is not linked properly, I was not able to check:-
- [ ] When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- [ ] When the user clicks (or taps) the close (X) button, the popup disappears.
- [ ] The popup is responsive, it fits mobile and desktop sizes properly.
- [ ] The popup follows the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colours, images, tests, margins).
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ONJoseph** [comment]
O, what an oversight! Thanks a lot @sja-thedude will
proceed to effect the changes immediately and resubmit for approval. Thanks
🙏👍👍

File: css/main.css [hunk 2]
```diff
@@ -458,6 +461,105 @@ body {
...
+
+.modal__feature ul li {
+  margin-left: 1.5rem;
+}
+
+.modal.active {
+  visibility: visible;
+  transform: scale(1);
+  opacity: 1;
+}
```
Comment (VICTOR-CHUKWU) [inline css/main.css]: - [x] Nice work so far, in your desktop view the modal is not fitted to the screen making it difficult for the user to see the close button. kindly fix
see image below 👇 
    
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @ONJoseph  👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/main.css (hunk 2 above)
Comment (ONJoseph) [reply css/main.css]: Thanks for the review .@VICTOR-CHUKWU! Please could use kindly provide helpful pointers to fix this issue? This is my last submission chance. Awaiting your response. Thanks.

Comment (ONJoseph) [reply css/main.css]: Have effected the change. Will proceed to submit for approval now. Thanks a lot! @VICTOR-CHUKWU 
Please can you kindly check and confirm it's ok now. This is my last submission chance please. Thanks.

**Review by JuliCarracedo — APPROVED** [review]
Hi @ONJoseph , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ONJoseph** [comment]
Hi@*JuliCarracedo, I'm so happy! Thanks a lot 👍👍*

**Pull request merged**